### PR TITLE
hosting.md: Add instructions for enabling Live Streaming

### PR DESCRIPTION
**Pull request by roberth**
Maybe it's because my google account is pretty old that I have to do this, idk. This has not been tested completely yet either; just sharing my observations and what I'm doing, as of writing.

### CODE REVIEW

**Comment by roberth** [comment]
Full text was

Request access to streaming
Once requested, it may take up to 24 hours to get access to live streaming
[REQUEST]

File: hosting.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ If you have one already you can use that one, but you're free to create a new on
 ### Get access to the YouTube channel
 
 - Send the E-Mail address of your Google account to [@infinisil:matrix.org](https://matrix.to/#/@infinisil:matrix.org)
+- Request Live Streaming for your Google account
```
Comment (roberth) [inline hosting.md]:
```suggestion
- Request Live Streaming for your Google account, 24h in advance
```

It's literally showing me a countdown for those 24 hours to pass, so be careful to prepare in advance!

**Comment by roberth** [comment]
> Maybe it's because my google account is pretty old

I had to verify my phone number, fwiw.

I can only guess what their policy is for the 24 hour wait. It could be unconditional, or my account might be an unlucky exception. [This document](https://support.google.com/youtube/answer/2474026?hl=en) states "may take up to 24 hours".

**Comment by roberth** [comment]
Also the whole reason for doing this is that jit.si reported "Live Streaming is not enabled for <my gmail>".
It did that both before I accepted the channel invitation and after, suggesting that this is a restriction on individual accounts, although I can't rule out a stale authorization cache, fwiw.

**Comment by infinisil** [comment]
Oh yeah that doesn't sound right. This would do live streaming on your own YouTube channel, not the NAT one. This was a mistake in the instructions, this should work instead: https://github.com/nixpkgs-architecture/.github/pull/6

**Comment by roberth** [comment]
:(

**Pull request closed without merging**